src/core/strategies/websocket_prioritized_cached_strategy.ts

```diff
@@ -47,7 +48,7 @@ export default class CachedStrategy implements Strategy {
     var strategies = [this.strategy];
     if (info && info.timestamp + this.ttl >= Util.now()) {
       var transport = this.transports[info.transport];
-      if (transport) {
+      if (transport && info.attempts && info.attempts > 3) {
```

Comment:
I'm hardcoding a few values here and below. I thought we should get feedback and then maybe later try to refactor it.